src/Miunie.WindowsApp/ViewModels/SettingsPageViewModel.cs

```diff
@@ -14,7 +14,7 @@ namespace Miunie.WindowsApp.ViewModels
 {
     public class SettingsPageViewModel : ViewModelBase
     {
-        private const string DefaultAvatarUrl = "../Assets/miunie-scarf-transparent.png";
+        private const string _defaultAvatarUrl = "../Assets/miunie-scarf-transparent.png";
```

Comment:
Our style for private **constants** actually is [Pascal case](http://wiki.c2.com/?PascalCase) so the previous was correct. 😊